libs/build-sqlcipher-desktop.sh

```diff
@@ -118,6 +124,41 @@ if [ -n "$CROSS_COMPILE_MACOS" ]; then
     CFLAGS="${CFLAGS} ${SQLCIPHER_CFLAGS} -I${OPENSSL_DIR}/include -L${OPENSSL_DIR}/lib" \
     LDFLAGS="${LDFLAGS} -L${OPENSSL_DIR}/lib" \
     LIBS="-lcrypto"
+elif [[ "$CROSS_COMPILE_TARGET" =~ "win32-x86-64" ]]; then
+
+pushd ..
+
+# From https://github.com/qTox/qTox/blob/9525505bff8719c84b6193174ea5e7ec097c54b8/windows/cross-compile/build.sh#L390-L446.
```

Comment:
Good lord this must have taken some digging.

Reply:
It did!  But it's easy to search for, since there are only a few folks trying.